crm_won_restrict_per_stage/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+    * Carolina Fernandez
```

Comment:
In general, when someone participates in the design, it's important to put them in the contributors. Decide if I have contributed enough for this module to put me here.